### ISSUE TITLE
⚙️ Task Mapping / Trouble with Taxonomies

### ISSUE THREAD

**Issue by xehu** [Watts-Lab/task-mapping#376]

**Comment by markwhiting** [comment]
Exciting. Looking forward to it!

**Comment by JamesPHoughton** [comment]
Happy to look at it =)
Are you getting chatGPT as a reviewer?

**Comment by xehu** [comment]
Haha @JamesPHoughton I have not given it to ChatGPT yet lol....human reviewers only for now!

**Comment by markwhiting** [comment]
Awesome! Excited for the paper. The timeline seems good.

**Comment by markwhiting** [comment]
Awesome! Will be feed backing shortly!

**Comment by markwhiting** [comment]
> I want to think about a good way to create a public-facing version of the repository, and make it easier for people to contribute their own tasks/dimensions (as we promised). Do people think it's a good idea to hire someone on Upwork and get them to help set up a front-end/back-end system?

I think theres a lot to be done for this as we have outlined, but we can do it after the paper submission. I think hiring someone on upward or an RA would be fine, but I think we should perhaps refine our design a bit before launching into that.

**Comment by xehu** [comment]
@markwhiting yes; the proposal here is that we figure out a bare-minimum/MVP release *now*, so that we can write about it in the paper that we will be sending to publication very soon. I don't think it should wait, because as @linneagandhi pointed out in her feedback this week, it's good to strike while the iron is hot; and to essentially create a mechanism for people to take action if they are inspired when they read the paper.

Thus, I'm wondering if you all have thoughts on some initial designs that we could get quickly implemented via Upwork?

**Comment by linneagandhi** [comment]
@xehu I wonder if maybe you could be even lighter with an MVP? Meaning, you could setup a public website that has a D3 style visualization of the current state tasks. (Something pretty with dimensions + images of the tasks). Then have a "contact us" section for people to submit ideas - and you could even have a second page on the site where you log those (maybe weekly, in batches) with credit (for incentive). That way the code involved would be super light touch. Another tool that an engineering friend suggested to me was ['retool](https://retool.com/) - but I haven't explored it in depth. He suggested it would be a super light way to spin up a backend/frontend with little technical knowledge reqd.

**Comment by markwhiting** [comment]
Sorry on my delay this week — aiming to provide more feedback tonight!
